docs/README.md

```diff
@@ -322,6 +338,7 @@ These common options are used in multiple APIs. Instead of repeating the same op
 | options.showRememberMe                                            | boolean           | false    | Value set during `init`.<p><strong>OR</strong></p>`true`  | If you wish to disable the `Stay logged in for 30 days` checkbox in the login screen, set this to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | options.isNewTab                                                  | boolean           | false    | Value set during `init`.<p><strong>OR</strong></p>`false` | Call method and open/render in a new browser tab, by default all views open in the same tab.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| options.authnMethod   | array of string with values `passwordless` or `sso` or string with value `passwordless` or `sso`    | `undefined`    | Value set during `init`.<p><strong>OR</strong></p>`undefined` | Switch between authentication methods version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
```suggestion
| options.authnMethod   | string <p><strong>OR</strong></p> string[] | | Value set during `init`. | Use different authentication methods. This can be a string or an array of the following values:<br /><br />`sso`, `passwordless`. |
```

Please stick to the existing document language or the way it explains for each column.

This options is also needed for some other APIs.

Reply:
we updated the docs to reflect the document language as you suggested here. What do you mean by `This option is also needed for some other APIs`??

Reply:
Nope, my mistake, I thought you only add it to one of the API, just scroll up and saw it is under common API, all good.